src/_includes/layouts/base.njk

```diff
@@ -101,6 +101,12 @@
                     <ul id="nav-content" class="hidden absolute top-8 right-0 border shadow-sm md:inline md:shadow-none w-36 z-10 md:border-0 bg-white md:bg-transparent md:w-auto rounded md:static md:float-none md:flex md:flex-row flex-col items-center justify-between font-medium text no-underline">
                         <li class="flex"><a href="/features">features</a></li>
                         <li class="flex"><a href="/pricing">pricing</a></li>
+                        <li class="ff-nav-dropdown relative">
+                            <a class="flex items-center gap-2">solutions {% include "components/icons/chevron-down.svg" %}</a>
```

Comment:
FYI this _could_ use the `{% image` helper, but we already have this convention in this file so no need to change it now